index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+  <header id="header">
+    <img id="header-img" src="https://unsplash.com/photos/CglmcwSuMXQ" alt="" />
+    <nav id="nav-bar">
+      <a class="nav-link" href="#header-img">Nav1</a>
+      <a class="nav-link" href="#header-img">Nav2</a>
+      <a class="nav-link" href="#header-img">Nav2</a>
+    </nav>
+  </header>
+  <div class="container">
+    <div>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
```

Comment:
I think it would be better to add the method attribute so that users specifies the HTTP method to be used when submitting the form data. and then use POST as the value in order to append the form data inside the body of the HTTP request.

Reply:
Well done! 😉. The idea is there but in the future make sure to double-check your answers or explanations so that they seem as clear as possible before giving them.